### PR TITLE
[BULK UPDATES] DocuTune - Fix build validation issues: docs-link-absolute

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: README.md [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
 ### \adapters-and-accelerators
 Contains all the installation and conceptual documentation for the BizTalk Adapter Pack (BAP), HL7, RosettaNet, SWIFT, FileAct/Interact, and the WCF LOB Adapter SDK. It does not include the built-in or native adapters, such as FTP, or WCF-WebHTTP. These are included in *\core*.
 
 ### \core
-Contains all the conceptual documentation, including getting started, tutorials, performance, the built-in adapters, B2B, and BAM. These topics typically apply to multiple BizTalk versions. 
+Contains all the conceptual documentation, including getting started, tutorials, performance, the built-in adapters, B2B, and BAM. These topics typically apply to multiple BizTalk versions.
 
 ### \dev-centers
-Contains all the stand-alone topics at [https://docs.microsoft.com/biztalk](https://docs.microsoft.com/biztalk), including the community-written blogs.
+Contains all the stand-alone topics in [Microsoft BizTalk Server documentation](https://docs.microsoft.com/biztalk)(, including the community-written blogs.
```
Comment (JustPies) [inline README.md:42]: An extra "(" was added that needs to be removed.

**Comment by JustPies** [comment]
#docutune-wait

File: README.md (hunk 1 above)
Comment (alexbuckgit) [reply README.md]: Fixed

**Comment by alexbuckgit** [comment]
#docutune-ready

**Review by MandiOhlinger — APPROVED** [review]

**Pull request merged**